lumen/ai/llm.py

```diff
@@ -564,3 +572,15 @@ def _get_delta(cls, chunk: Any) -> str:
     def _add_system_message(self, messages: list[Message], system: str, input_kwargs: dict[str, Any]):
         input_kwargs["system"] = system
         return messages, input_kwargs
+
+
+class AINavigator(OpenAI):
+    """
+    A LLM implementation that calls the [Anaconda AI Navigator](https://www.anaconda.com/products/ai-navigator) API.
+    """
+
+    endpoint = param.String(
+        default="http://127.0.0.1:8080/v1", doc="""
```

Comment:
Prefer the more readable version:

```suggestion
        default="http://localhost:8080/v1", doc="""
```